### DISCUSSION TITLE
nuxsmin still active on this project?

### DISCUSSION THREAD

**Discussion by bitking** [nuxsmin/sysPass#1822] [Q&A]
Hi there, i really hope nuxsmin is still active on this project.

I use this software for a lot of years with pleasure thanks to him.

The last time there was a commit was on 13 march 2021.

Can anyone explain what is the current status?

Or @nuxsmin Can you elaborate?

**Comment by nuxsmin** [comment]
Sure @bitking I'm here.

Last year was quite intense (job change, etc.), though I started a few useful features, like the CLI module and refactoring the code to implement PHP 8 features. Right now I'm setting GitHub actions to run the CI workflow.

Regards!
